types/wkhtmltopdf/wkhtmltopdf-tests.ts

```diff
@@ -0,0 +1,37 @@
+import wkhtmltopdf from "wkhtmltopdf";
```

Comment:
```suggestion
import wkhtmltopdf = require('wkhtmltopdf');
```
as per readme:
var wkhtmltopdf = require('wkhtmltopdf');

Reply:
Thanks, I didn't know about this syntax. I've read about it and this makes way more sense :+1: